v2/cmd/wails/internal/commands/generate/template/base/frontend/dist/index.html

```diff
@@ -1,18 +1,20 @@
+<!DOCTYPE html>
 <html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/main.css" />
+  </head>
 
-<head>
-  <link rel="stylesheet" href="/main.css">
-</head>
-
-<body data-wails-drag>
-  <div class="logo"></div>
-  <div class="result" id="result">Please enter your name below 👇</div>
-  <div class="input-box" id="input" data-wails-no-drag>
-    <input class="input" id="name" type="text" autocomplete="off">
-    <button class="btn" onclick="greet()">Greet</button>
-  </div>
-
-  <script src="/main.js"></script>
-</body>
-
-</html>
\ No newline at end of file
+  <body data-wails-drag>
```

Comment:
We should remove all the drag attributes from the default template. People have reported issues with click events not realising this attribute will swallow them.

Reply:
If this property is removed, how can the user drag the window in borderless mode?

Reply:
By adding it back in? 😄